src/utils.js

```diff
@@ -57,10 +61,37 @@ export function toWidget( element, options = {} ) {
 		setLabel( element, options.label );
 	}
 
+	setVirtualSelectionHandling( element, setVirtualSelection, removeVirtualSelection );
```

Comment:
I don't like the fact that `setVirtualSelection` and `removeVirtualSelection` are called the same as widget methods, it's misleading. Can you just use arrow function?  Or it could be a default value of `setVirtualSelectionHandling`.